manager/manager.go

```diff
@@ -30,6 +32,7 @@ func NewManager(cacheRoot string, configFile string, strict bool, db *gorm.DB) *
 		configFile: configFile,
 		strict:     strict,
 		db:         db,
+		uuid:       tracking.LoadRancherUUID(),
```

Comment:
Can `uuid` be an arg to `NewManager`? Then `tracking.LoadRancherUUID` can be called just by `main()`.